watertap/costing/watertap_costing_package.py

```diff
@@ -1058,3 +1100,35 @@ def cost_by_flow_volume(blk, flow_cost, flow_to_cost):
     blk.capital_cost_constraint = pyo.Constraint(
         expr=blk.capital_cost == blk.flow_cost * flow_to_cost
     )
+
+
+def cost_uv_aop_stack(blk, flow_cost, electricity_cost, factor_uv_replacement):
+    """
+    Generic function for costing by flow volume.
+
+    Args:
+        flow_cost - The cost of UV reactor in [currency]/[volume]
+        electricity_cost - The costs of the lamps, sleeves, ballasts and sensors in [currency]/[kW]
+    """
+    make_capital_cost_var(blk)
+    make_fixed_operating_cost_var(blk)
+    blk.flow_cost = pyo.Expression(expr=flow_cost)
+    blk.electricity_cost = pyo.Expression(expr=electricity_cost)
+    blk.factor_uv_replacement = pyo.Expression(expr=factor_uv_replacement)
+
+    reactor_volume = pyo.units.convert(
+        blk.unit_model.reactor_volume, to_units=pyo.units.m**3
+    )
+
+    electricity_demand = pyo.units.convert(
+        blk.unit_model.electricity_demand[0], to_units=pyo.units.kW
+    )
+
+    blk.capital_cost_constraint = pyo.Constraint(

```

Comment:
Little confused with how `blk.electricity_cost * electricity_demand` is a capital cost, just thinking that some electricity demand in kW (or GJ/yr or any equivalent) should be a fixed operating cost. Are you costing the uv (lamp?) replacement as a function of the electricity demand? If so, why is the equation not something similar to:
`blk.fixed_operating_cost_constraint = pyo.Constraint(
        expr=blk.fixed_operating_cost
        == blk.factor_uv_replacement * blk.uv_replacement_cost * electricity_demand + blk.electricity_cost * electricity_demand
    )`